documentation/quarkus/guide-beanmapping-quarkus.asciidoc

```diff
@@ -0,0 +1,224 @@
+:toc: macro
```

Comment:
I'm not sure if this guide should be stored in the folder with Quarkus specific stuff, as this is not really Quarkus specific. You can use MapStruct exactly this way within spring applications. So maybe the better way would be to change the original bean mapping guide. But then we would have three different frameworks within this guide (Orika, Dozer, MapStruct).
The original guide has to be changed anyway as it depends on devon4j-spring specific modules ([devon4j-beanmapping](https://github.com/devonfw/devon4j/tree/master/modules/beanmapping)).

So my preferred solution would be to have one general guide explaining bean mapping, Orika and MapStruct in general (Dozer is already not recommended) then move the parts which are devon4j-spring specific (depending on devon4j-beanmapping) into a separate guide.
@SchettlerKoehler @hohwille What do you think?

Reply:
Sounds good

Reply:
I agree. We should have separate guides for bean mapping based on technology. IMHO dozer and orika can be addressed in the same guide describing how to use generic bean mapper via devon4j `BeanMapper` facade/shim. For MapStruct a totally different approach is needed. The existing `guide-beanmapping.asciidoc` can then only explain the basics and link to the different options.